tests/test_emulations.py

```diff
@@ -0,0 +1,50 @@
+import warnings
+from typing import List, Type
+
+import pytest
+
+from swat.commands.base_command import BaseCommand
+from swat.commands.emulate import Command as emulate_command
```

Comment:
Do you need this alias?

Reply:
`Command` is abstracted everywhere, so it definitely provides clarity to have it